public/.well-known/assetlinks.json

```
@@ -0,0 +1,10 @@
+[{
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+        "namespace": "android_app",
+        "package_name": "app.thelist.twa",
+        "sha256_cert_fingerprints": [
+        "7E:F9:82:82:6A:76:FE:99:E0:EC:AD:68:0C:10:32:B1:4E:06:B3:00:A0:96:45:4C:0E:C2:FF:5F:04:44:88:47"
```

Minor point, but the indentation doesn't seem right here 
```suggestion
            "7E:F9:82:82:6A:76:FE:99:E0:EC:AD:68:0C:10:32:B1:4E:06:B3:00:A0:96:45:4C:0E:C2:FF:5F:04:44:88:47"
```